config/express.js

```diff
@@ -22,7 +22,7 @@ var verifyHandler = function (adminToken, token, tokenSecret, profile, done) {
             redToken: tokenSecret,
             provider: profile.provider,
             name: profile.name,
-            flair: {ptrades: flair1, svex: flair2}
+            flair: {ptrades: flairs[0], svex: flairs[1]}
```

Comment:
It seems to be a bigger issue than that -- I can't get `grunt eslint` to find any errors at all, even when I randomly delete semicolons or add gibberish to controller files.